app/models/concerns/account_interactions.rb

```diff
@@ -66,8 +74,12 @@ def follow_mapping(query, field)
     has_many :domain_blocks, class_name: 'AccountDomainBlock', dependent: :destroy
   end
 
-  def follow!(other_account)
-    active_relationships.find_or_create_by!(target_account: other_account)
+  def follow!(other_account, reblogs: nil)
+    reblogs = true if reblogs.nil?
```

Comment:
Can this not be part of the argument's default value..?

Reply:
In fact no, apparently. (Well, it could be, but we still need to catch the nil case: it may come in from params as nil.)